schema/handler.go

```diff
@@ -0,0 +1,46 @@
+package schema
+
+import (
+	"github.com/julienschmidt/httprouter"
+	"github.com/ory/kratos/x"
+	"net/http"
+)
+
+type (
+	Configuration       interface{}
+	handlerDependencies interface {
+		PoolProvider
+		x.WriterProvider
+	}
+	HandlerProvider interface {
+		IdentityHandler() *Handler
+	}
+	Handler struct {
+		c Configuration
+		r handlerDependencies
+	}
+)
+
+func NewHandler(
+	c Configuration,
+	r handlerDependencies,
+) *Handler {
+	return &Handler{
+		c: c,
+		r: r,
+	}
+}
+
+func (h *Handler) RegisterPublicRoutes(public *x.RouterPublic) {
+	public.GET("/schemas/:id", h.get)
+}
+
+func (h *Handler) get(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
+	s, err := h.r.SchemaPool().GetSchema(x.ParseUUID(ps.ByName("id")))
+	if err != nil {
+		h.r.Writer().WriteError(w, r, err)
+		return
+	}
+
+	h.r.Writer().Write(w, r, s)
```

Comment:
I think we should only respond with the actual schema content, not the wrapper? So probably something like `Write(w, r, s.Schema)`

Reply:
So I just saw that we don't have the schema yet - only it's url. We should probably get that somewhere. Either here (with caching), or store it in the database directly. In both cases we need a strategy to update the schema if it changes - but we don't need to do that now (but please create an issue if you skip the refresh here).